test/Npgsql.CrateDb.Tests/CrateDbTypesTest.cs

```diff
@@ -0,0 +1,485 @@
+            }
+        }
+
+        [Test]
+        public void TestSelectIntegerType()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = new NpgsqlCommand("select integer_field from test", con))
+            {
+                var r = cmd.ExecuteScalar();
+                Assert.That(r, Is.EqualTo(1200000));
+            }
+        }
+
+        [Test]
+        public void TestSelectLongType()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = new NpgsqlCommand("select long_field from test", con))
+            {
+                var r = cmd.ExecuteScalar();
+                Assert.That(r, Is.EqualTo(120000000000L));
+            }
+        }
+
+        [Test]
+        public void TestSelectFloatType()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = new NpgsqlCommand("select float_field from test", con))
+            {
+                var r = cmd.ExecuteScalar();
+                Assert.That(r, Is.EqualTo(1.4f));
+            }
+        }
+
+        [Test]
+        public void TestSelectDoubleType()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = new NpgsqlCommand("select double_field from test", con))
+            {
+                var r = cmd.ExecuteScalar();
+                Assert.That(r, Is.EqualTo(3.456789d));
+            }
+        }
+
+        [Test]
+        public void TestSelectTimestampType()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = new NpgsqlCommand("select timestamp_field from test", con))
+            {
+                var r = cmd.ExecuteScalar();
+                Assert.That(r, Is.EqualTo(new DateTime(2018, 10, 11).ToUniversalTime()));
+            }
+        }
+
+        [Test]
+        public void TestSelectIPType()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = new NpgsqlCommand("select ip_field from test", con))
+            {
+                var r = cmd.ExecuteScalar();
+                Assert.That(r, Is.EqualTo("127.0.0.1"));
+            }
+        }
+
+        [Test]
+        public void TestSelectGeoPoint()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = new NpgsqlCommand("select geo_point_field from test", con))
+            {
+                var r = cmd.ExecuteScalar();
+                Assert.That(r, Is.EquivalentTo(new double[] { 9.7419021d, 47.4048045d }));
+            }
+        }
+
+        [Test]
+        public void TestSelectGeoShape()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = new NpgsqlCommand("select geo_shape_field from test", con))
+            {
+                var r = cmd.ExecuteScalar();
+                Assert.That(r, Is.InstanceOf(typeof(string)));
+
+                var gs = Newtonsoft.Json.JsonConvert.DeserializeObject<GeoShape>(r.ToString());
```

Comment:
nit: is full qualification required here?

Reply:
no removed it on all places in this class. commit will follow